mk/mcu/STM32F4.mk

```diff
@@ -100,7 +100,8 @@ endif
 VPATH           := $(VPATH):$(CMSIS_DIR)/Core/Include:$(ROOT)/lib/main/STM32F4/Drivers/CMSIS/Device/ST/STM32F4xx
 
 INCLUDE_DIRS    := $(INCLUDE_DIRS) \
-                   $(ROOT)/src/main/drivers/stm32
+                   $(SRC_DIR)/startup/stm32 \
```

Comment:
Maybe move path to #include (this global include path is used only for few files)

Reply:
Required for inclusion from within the STM library.